packages/reakit/src/Tabs/TabsContainer.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import { ComposableContainer } from "constate";
+import StepContainer, {
+  StepContainerSelectors,
+  StepContainerActions,
+  StepContainerState
+} from "../Step/StepContainer";
+
+interface TabsContainerInitialState {
+  loop: boolean;
+  current: number;
+}
+
+export const initialState: TabsContainerInitialState = {
+  loop: true,
+  current: 0
+};
+
+// istanbul ignore next
+const TabsContainer: ComposableContainer<
+  StepContainerState,
+  StepContainerActions,
+  StepContainerSelectors
+> = props => (
```

Comment:
Nop, it can't find initialState on 

```js
Pick<ContainerProps<StepContainerState, StepContainerActions, StepContainerSelectors, {}>, "initialState" | "context" | "onMount" | "onUpdate" | "onUnmount" | "shouldUpdate" | "children">
```

Which doesn't make sense since `ContainerProps<StepContainerState, StepContainerActions, StepContainerSelectors, {}>` has initialState.

Any idea?

Reply:
That's weird. Well, I think we can leave it as it is for now.